.travis.yml

```diff
@@ -21,5 +21,5 @@ script:
   - sh -x ./node_modules/patternfly-eng-release/scripts/_build.sh -o
 
 after_success:
-  - grunt build
+  - grunt build:production
```

Comment:
The build script from patternfly-eng-release also performs a build. Basically, we're building the repo twice. Do we want the patternfly-eng-release script to use grunt build:production instead?